Formula/openssl@3.rb

```diff
@@ -1,9 +1,9 @@
 class OpensslAT3 < Formula
   desc "Cryptography and SSL/TLS Toolkit"
   homepage "https://openssl.org/"
-  url "https://www.openssl.org/source/openssl-3.0.6.tar.gz"
-  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-3.0.6.tar.gz"
-  sha256 "e4a10a2986945e3f1a1f2ebd68ac780449a1773b96b6a174fdf650d6bc9611f1"
+  url "https://www.openssl.org/source/openssl-3.0.5.tar.gz"
+  mirror "https://www.mirrorservice.org/sites/ftp.openssl.org/source/openssl-3.0.5.tar.gz"
+  sha256 "aa7d8d9bef71ad6525c55ba11e5f4397889ce49c2c9349dcea6d3e4f0b024a7a"
   license "Apache-2.0"
```

Comment:
We probably need a version_scheme as this is a downgrade
```suggestion
  license "Apache-2.0"
  version_scheme 1
```

Reply:
We've not done so for the other version: https://github.com/Homebrew/homebrew-core/pull/112984
It's an emergency measure, not a proper version change. We just want to prevent more people from upgrading over time.